spec/Section 6 -- Execution.md

```diff
@@ -224,9 +224,10 @@ must receive no more events from that event stream.
 
 **Supporting Subscriptions at Scale**
 
-Supporting subscriptions is a significant change for any GraphQL service. Query
-and mutation operations are stateless, allowing scaling via cloning of GraphQL
-service instances. Subscriptions, by contrast, are stateful and require
+Supporting subscriptions at scale for a GraphQL service requires taking into
+consideration the need to preserve state on the server. 
+Query and mutation operations are stateless, allowing scaling via cloning 
+of GraphQL service instances. Subscriptions, by contrast, are stateful and require
```

Comment:
This seems redundant ("subscriptions [...] need to preserve state on the server" vs "Subscriptions [...] are stateful"); perhaps we can just remove the first sentence entirely:

```suggestion
Query and mutation operations are stateless, allowing scaling via cloning 
of GraphQL service instances. Subscriptions, by contrast, are stateful and require
```

Reply:
agree, let's just remove it